.gitignore

```diff
@@ -26,13 +26,13 @@ bld/
 
 # Visual Studio 2015/2017 cache/options directory
 .vs
-.vscode
+.vscode
```

Comment:
I assume I removed trailing spaces here, but anyone know how to validate? GitHub doesn't seem to have the option to show whitespace, and in particular whitespace changes

Reply:
Well, at least you can hide whitespace changes...